trashtalk/apps/cleanups/models.py

```diff
@@ -20,7 +20,7 @@ class Cleanup(models.Model):
     host = models.ForeignKey(User, on_delete=models.CASCADE, related_name="cleanups")
     location = models.ForeignKey('Location', on_delete=models.CASCADE)
 
-    participants = models.ManyToManyField(User)
+    participants = models.ManyToManyField(User, related_name="cleanups_participated")
```

Comment:
You've got the right idea, but this relation already exists. You can currently use `User.cleanup_set.all()`. Go ahead and test this out for yourself in the python interpreter: `python manage.py shell_plus` will load all the application models and you can query any user.
```
user = User.objects.get(id=1) # or whichever particular user you want to try this with
user.cleanup_set.all()
```

However, I had some technical difficulties last night getting the tests to work so just did the query manually in view. I'll be having another look tonight at what was going there.

Reply:
@Protosac that makes sense.